Project-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,17 @@
 import React from 'react';
 import './Card.css';
+
+const CardContent = () => {
+    return (
+        <span>
```

Comment:
A *div* is more appropriate here as *h3* and *p* elements are block elements and *span* is inline.